birth_test.rb

```diff
@@ -0,0 +1,20 @@
+expected = <<-'END'
```

Comment:
I think this file was committed by accident?

Reply:
Definitely was, I'll just force push it out of the commit.